src/www/ui/admin-dashboard.php

```diff
@@ -226,9 +226,11 @@ function DatabaseQueries()
     $head3 = _("Started");
     $head4 = _("Elapsed");
     $V .= "<tr><th>$head1</th><th>$head2</th><th>$head3</th><th>$head4</th></tr>\n";
-
-    $current_query = (strcmp($this->pgVersion['server'], "9.2") >= 0) ? "state" : "current_query";
-    $procpid = (strcmp($this->pgVersion['server'], "9.2") >= 0) ? "pid" : "procpid";
+    $getCurrentVersion = explode(" ", $this->pgVersion['server']);
+    $currentVersion = str_replace(".", "", $getCurrentVersion[0]);
+    unset($getCurrentVersion);
+    $current_query = ($currentVersion >= "92") ? "state" : "current_query";
```

Comment:
better to define the version("9.2") as a variable instead of "92" for better understanding.